charter.md

```diff
@@ -72,28 +78,156 @@ this works.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected steering committee
-member, the candidate with the next most votes from the previous election will
-be offered the seat.  This process will continue until the seat is filled.
+In the event of a resignation or other loss of an elected committee member, the
+next most preferred candidate from the previous election will be offered the
+seat.
+
+A maximum of one (1) committee member may be selected this way between
+elections.
 
 In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. [Eligible voters][voter-eligibility] from the most
-recent election will vote in the special election i.e., eligibility will not be
-redetermined at the time of the special election. A committee member elected in
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+Examples (non-exhaustive):
+
+* the next most preferred candidate is no longer interested in sitting on the
+  committee
+* the next most preferred candidate cannot sit on the committee, due to some
+  committee composition requirement i.e.,
+  [maximal representation][maximal-representation]
 
+[Eligible voters][voter-eligibility] from the most recent election will vote in
+the special election i.e., eligibility will not be redetermined at the time of
+the special election.
+
+A committee member elected in a special election will serve out the remainder
+of the term for the person they are replacing, regardless of the length of that
+remainder.
+
+[maximal-representation]: /elections.md#maximal-representation
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee
+meeting and must be documented as a GitHub issue in the committee's
+[repository][steering-repo].
+
+The call for a vote of no confidence must be made by a current member of the
+committee and must be seconded by another current member.
+
+The committee member who calls for the vote will prepare a short statement
```

Comment:
do we need to define "short" ?

Reply:
if we want to pick, it would be 300 words as the bio for elections ( https://github.com/kubernetes/community/search?q=300+words ), we could say approximately 300 words if we insist on defining "short". 

BUT i don't see the necessity to define short here. we are just telling people that it wont be a really long thing i feel.

Reply:
This would only be executed under potentially very antagonistic conditions and IMO "short" needs to be defined. Otherwise there will be a fight about whether the board's actions were valid under the board's own rules.

Reply:
I think we should remove the word short and leave the rest

Reply:
> I think we should remove the word short and leave the rest

Agreed, for simplicity. Removed in https://github.com/kubernetes/steering/pull/248/commits/d4552c5014a97cf6983c2fd85d50b0bb8eeb0b88.